dev-client/src/screens/SoilScreen/components/SoilDepthIntervalSummary.tsx

```diff
@@ -59,7 +59,10 @@ const DepthIntervalEditor = ({
       py="8px">
       <Heading variant="h6" color="primary.contrast">
         {interval.label && `${interval.label}: `}
-        {t('soil.depth_interval.bounds', {...interval, units: 'cm'})}
+        {t('soil.depth_interval.bounds', {
+          ...interval.depthInterval,
+          units: 'cm',
```

Comment:
question: should this come from a setting?

Reply:
yes, but that setting doesn't exist yet and making that change is out of scope for this PR. tracked by https://github.com/techmatters/terraso-mobile-client/issues/804

Reply:
Please add a TODO here so we don't forget.